CONTRIBUTING.md

```diff
@@ -164,6 +165,8 @@ Make sure that [Travis](http://www.travis-ci.org) greenlights the pull request w
  - newline between } and else instead of } else {
  - if one part of an if..else chain uses { } then all should
  - no whitespace at the end of a line
+ - no indentation for macros like #if and #ifdef
```

Comment:
Oh, I didn't know that preprocessor directives and macros are not the same. I'll change that now!